### PR TITLE
Add query and update animals table

### PR DESCRIPTION
**Pull request by FuadNabi**
PullRequestGIF

#In This Pull Request:🎉🚀🌐

## Screenshots

- Screenshot 2023-01-24 152602
- Screenshot 2023-01-24 152806
- Screenshot 2023-01-24 153014
- Screenshot 2023-01-24 153409
- Screenshot 2023-01-24 154305
- Screenshot 2023-01-24 154452
- Screenshot 2023-01-24 154753
- Screenshot 2023-01-24 155833
- Screenshot 2023-01-24 160259
- Screenshot 2023-01-24 160413
- Screenshot 2023-01-24 161053
- Screenshot 2023-01-24 161144
- Screenshot 2023-01-24 161322
- Screenshot 2023-01-24 161527
- Screenshot 2023-01-24 161953
- Screenshot 2023-01-24 162237
- Screenshot 2023-01-24 162910
- Screenshot 2023-01-24 163756

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -13,3 +13,55 @@ SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+  UPDATE animals
+  SET species = 'unspecified';
+
+ROLLBACK;
```
Comment (Munsa1) [inline queries.sql]: - [ ] Before and after rolling back changes kindly add a select statement for `species` to check if changes were made. So add one before and another after the `Rollback;` statement


`SELECT species from animals;`

File: queries.sql [hunk 2]
```diff
@@ -13,3 +13,55 @@ SELECT * FROM animals WHERE neutered IS true;
...
+BEGIN;
+
+  UPDATE animals 
+  SET species = 'digimon'
+  WHERE name LIKE '%mon%';
+
+  UPDATE animals
+  SET species = 'pokemon'
+  WHERE species is NULL;
+
```
Comment (Munsa1) [inline queries.sql:38]: - [ ] Again here kindly add a select statement  to check if changes were made before `commit`.

File: queries.sql [hunk 3]
```diff
@@ -13,3 +13,55 @@ SELECT * FROM animals WHERE neutered IS true;
...
+COMMIT;
+
```
Comment (Munsa1) [inline queries.sql:42]: - [ ] Again here kindly add a select statement  to check if changes persist after `commit` is made.

File: queries.sql [hunk 4]
```diff
@@ -13,3 +13,55 @@ SELECT * FROM animals WHERE neutered IS true;
...
+BEGIN;
+  TRUNCATE TABLE animals;
+ROLLBACK;
+
```
Comment (Munsa1) [inline queries.sql]: - [ ] Again here kindly add a select statement  to check if changes were made after the `truncate` statement and after the `rollback` statement.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @FuadNabi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @FuadNabi, 

To highlight 👍🏽 
✔️ Queries are successful
✔️ You added screenshots

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**